addons/website_sale/models/product_wishlist.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from odoo.http import request
+
+
+class ProductWishlist(models.Model):
+    _name = 'product.wishlist'
+    _description = 'Product Wishlist'
+    _sql_constraints = [
+        ("product_unique_partner_id",
+         "UNIQUE(product_id, partner_id)",
+         "Duplicated wishlisted product for this partner."),
+    ]
+
+    partner_id = fields.Many2one('res.partner', string='Owner')
+    product_id = fields.Many2one('product.product', string='Product', required=True)
+    currency_id = fields.Many2one('res.currency', related='pricelist_id.currency_id', readonly=True)
+    pricelist_id = fields.Many2one('product.pricelist', string='Pricelist', help='Pricelist when added')
+    price = fields.Monetary(currency_field='currency_id', string='Price',
+                            help='Price of the product when it has been added in the wishlist')
+    website_id = fields.Many2one('website', ondelete='cascade', required=True)
+    active = fields.Boolean(default=True, required=True)
+    stock_notification = fields.Boolean(default=False, required=True)
+    displayed_in_cart = fields.Boolean(default=True, required=True,
+                                       help='Is used to tell whether a wishlist record should be displayed in the '
+                                            'user wishlist')
```

Comment:
If it's about being visible in the wishlist why is it called "visible_*in_cart*"?

Surely it should be `visible_in_wishlist`? Or given it's a wishlist item and there's nowhere else it *could* be displayed, why not just "visible" or "displayed"?